example.rst

```diff
@@ -0,0 +1,37 @@
+Training example
+================
+
+In this document, we go through a common workflow that a scientist wishing to use OMERO will follow:
+
+ * Import data using the Desktop client.
+ * View images using the OMERO.iviewer plugin
+ * Analyze images using a 3rd party tool e.g. Fiji.
+ * Generate a figure using OMERO.figure.
+
+
+:doc:`upload/docs/import-desktop-client`
```

Comment:
The rendering of these is odd. The link is small then "Description" is large. Is it true that these are subdocuments that are also imported elsewhere? If so, I'd make that clear and perhaps this would then fit under `how to write a guide` since one could imagine others.

Reply:
The rendering is b/c Description is in bold.
We can sort that out

Reply:
This is no longer bold so it should render correctly now

Reply:
This is no longer bold so it should render correctly now

Reply:
Those subsections "Description" were not used until now. 
This is a proposal following @will-moore comments. If we are happy with this, we can clarify it as you suggested.

Reply:
I just built this locally using 7a79d0d and I'm still seeing the 'bold' Descriptions and small links:

![Screenshot 2020-05-19 at 11 59 23](https://user-images.githubusercontent.com/900055/82318831-6b3ead00-99c8-11ea-9b7d-c2098a15ebb7.png)

Reply:
Yes. The change is in https://github.com/ome/omero-guide-upload

Reply:
I am using ``:doc:`upload/docs/import-desktop-client```` to extract the title.
Need to see how to make it bigger